Server/EVESharp.Node/Services/Dogma/dogmaIM.cs

```diff
@@ -254,15 +256,11 @@ protected override BoundService CreateBoundInstance (ServiceCall call, ServiceBi
         if (this.MachoResolveObject (call, bindParams) != BoundServiceManager.MachoNet.NodeID)
             throw new CustomError ("Trying to bind an object that does not belong to us!");
 
-        // make sure the character is loaded
```

Comment:
these comments should be kept as they explain a bit the rationale behind the code